pkg/cmd/operator/webhooks.go

```diff
@@ -228,17 +232,18 @@ func validate(ar *admissionv1.AdmissionReview) error {
 
 	switch gvr {
 	case scyllav1.GroupVersion.WithResource("scyllaclusters"):
+		var allErrs field.ErrorList
```

Comment:
nit : errs (allErrs kind of implies you'd be placing multiple errs into it)

Reply:
fixed